wigwam/cli/build_commands.py

```diff
@@ -139,10 +140,47 @@ def init_build_parsers(subparsers: argparse._SubParsersAction) -> None:
     )
     add_tag_argument(parser=install_parser, default="installed")
 
+    distrib_parser = subparsers.add_parser(
+        "make-distributable",
```

Comment:
Let's call this command `make-distrib` in order to cut down on verbosity a bit.

```suggestion
        "make-distrib",
```